manage-images.md

```diff
@@ -0,0 +1,120 @@
+An image is the base for a container running in the Anbox Cloud. It contains all necessary components, like Anbox or the Android root file system. Each release of Anbox Cloud comes with an update image, which must be uploaded to the AMS service to make it available to be used inside any container.
```

Comment:
Images are automatically downloaded by AMS if the image server is configured (which we can assume it always is). The `which must be uploaded to the AMS service` sounds like the user has to do some manual stuff IMO

Reply:
Seems there's quite a bit of outdated information in here. I understood it as if users had to do something - will update.